buddyboss/buddyboss-platform
# Fixed - Incorrect My Groups count when viewed on Group Type page

### All Submissions:






### Changes proposed in this Pull Request:

I have fixed three issues that are mentioned here: https://github.com/buddyboss/buddyboss-platform/issues/1743#issuecomment-780481003


Fixes #1743 .

### How to test the changes in this Pull Request:

1. Create a Group Type ex: Group Type A
2. Create at least two Groups, and associate one group with Group Type A
3. As a user, join those Groups
4. Now Group count, Group Tab, and Grid view and List view settings should be working.

### Proof Screenshots or Video

https://www.loom.com/share/7be212383676433aa99d450eab5f4a8d



### Other information:
